### PR TITLE
Assignments security

### PR DESCRIPTION
**Pull request by sandramchung**
In testing this tiny change, I realized that I don't actually know which roles should have access to assignments. I made some assumptions about contacts. But in looking at the user_organization table in the db, the roles are set per organization, not per campaign like I assumed, and I don't know what their UX is supposed to look like. So marking this as WIP until we get some clarity.

### CODE REVIEW

**Comment by mathemagica** [comment]
As an organizational admin, I should be able to assign contacts to campaigns. As a texter, I should not be allowed to view assignment screens or assign contacts to campaigns.

File: src/server/api/schema.js [hunk 1]
```diff
@@ -690,12 +708,16 @@ const rootResolvers = {
       userRoles.forEach(role => {
         roles[role['role']] = 1
       })
+<<<<<<< HEAD
```
Comment (shakalee14) [inline src/server/api/schema.js]: some merging issues here.

**Review by shakalee14 — APPROVED** [review]

**Comment by shakalee14** [comment]
@sandramchung can you handle this conflict?

**Comment by sandramchung** [comment]
@shakalee14 done

**Pull request merged**